Fix for unsupported scenario (#76632)
Azure/azure-content: articles/virtual-desktop/app-attach-msixmgr.md

msixmgr.exe -Unpack -packagePath "C:\Users\ssa\Desktop\packageName_3.51.1.0_x64__81q6ced8g4aa0.msix" -destination "c:\temp\packageName.vhdx" -applyacls -create -vhdSize 200 -filetype "vhdx" -rootDirectory apps
```

>[!NOTE]

> > Please avoid package names longer than 128 characters and inclusion of space in the MSIX image name.

Reviewer: ```suggestion
>This command doesn't support package names that are longer than 128 characters or MSIX image names with spaces between characters.
```
